### PR TITLE
Fix alpine image

### PR DESCRIPTION
**Pull request by zhouzhuojie**
Fixes dockerhub rate limiting issue, use the ECR image instead.

### CODE REVIEW

File: .github/templates/linux_ci_workflow.yml.in [hunk 1]
```diff
@@ -35,7 +35,7 @@ jobs:
       - name: Chown workspace
         run: |
           # Ensure the working directory gets chowned back to the current user
-          docker run --rm -v "$(pwd)":/v -w /v alpine chown -R "$(id -u):$(id -g)" .
+          docker run --rm -v "$(pwd)":/v -w /v "${ALPINE_IMAGE}" chown -R "$(id -u):$(id -g)" .
```
Comment (zhouzhuojie) [inline .github/templates/linux_ci_workflow.yml.in:38]: ALPINE_IMAGE was defined at the beginning of the file

**Review by samestep — APPROVED** [review]
lgtm, thanks! Sorry for not doing this initially; I didn't realize the purpose of `"${ALPINE_IMAGE}"`. Could you add a code comment stating that the purpose is to avoid rate limiting (perhaps linking to this PR)?

File: .github/templates/linux_ci_workflow.yml.in (hunk 1 above)
Comment (walterddr) [reply .github/templates/linux_ci_workflow.yml.in:43]: could you comment on the `env:` section where ALPINE_IMAGE were defined?

Comment (zhouzhuojie) [reply .github/templates/linux_ci_workflow.yml.in:43]: @samestep @walterddr Of course, will unblock first and save a new run of CI and do the comments in the following PRs

File: .github/templates/linux_ci_workflow.yml.in [hunk 2]
```diff
@@ -32,10 +32,15 @@ jobs:
     outputs:
       docker_image: ${{ steps.calculate-tag.outputs.docker_image }}
     steps:
+      - name: Log in to ECR
+        run: |
+          aws ecr get-login --no-include-email --region us-east-1 > /tmp/ecr-login.sh
```
Comment (malfet) [inline .github/templates/linux_ci_workflow.yml.in:37]: Please use `aws ecr get-login-password`, as get-login was removed in awscli-2.x 
```suggestion
          aws ecr get-login-lassword --region us-east-1| docker --username AWS --password-stdin "308535385114.dkr.ecr.us-east-1.amazonaws.com
```

Comment (zhouzhuojie) [reply .github/templates/linux_ci_workflow.yml.in:37]: sure, also going to refactor (it was duplicated in multiple places) and add a few code comments in the following PR

**Pull request closed without merging**